core/common/src/test/java/alluxio/wire/WorkerInfoTest.java

```diff
@@ -37,6 +40,23 @@ public void thrift() {
     checkEquality(workerInfo, other);
   }
 
+  @Test
+  public void checkLastContactSecComparator() {
```

Comment:
usually we don't prepend or append "test" or "check" to our tests because they are already marked with `@Test`